docs/user-guide.md

```diff
@@ -322,7 +322,23 @@ spec:
   sourceNamespace: openshift-marketplace
 ```
 
-Note: each file should contain a single object.
+#### Add required CRs
+
+If a CustomResource should be applied post operator installation, add the relevant CR to `${APPLIANCE_ASSETS}/openshift/crs`.
+The CRs will be applied automatically during cluster installation.
+
+E.g. a CR for `kubevirt-hyperconverged` operator:
+
+*openshift/crs/cnv_cr.yaml*
+```yaml
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+  namespace: "openshift-cnv"
+spec:
+  BareMetalPlatform: true
```

Comment:
Note: this field is currently not supported for the CR

Reply:
oh, it's used by the service: https://github.com/openshift/assisted-service/blob/f8a03caa9ff758b08b7590a31aecc77e083fb157/internal/operators/cnv/manifest.go#L193
Is it not supported in new versions?